common/patches/rof2.cpp

```diff
@@ -5638,7 +5699,13 @@ namespace RoF2
 		RoF2::structs::ItemBodyStruct ibs;
 		memset(&ibs, 0, sizeof(RoF2::structs::ItemBodyStruct));
 
-		ibs.id = item->ID;
+		if (item->ID == PARCEL_MONEY_ITEM_ID) {
```

Comment:
Declare the default case above and remove the else

```cpp
ibs.id = item->ID;
if (item->ID == PARCEL_MONEY_ITEM_ID) {
	ibs.id = inst->GetSerialNumber();
}
```

Reply:
Resolved